pkg/cassandra/config/schema_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestQueryCreationFromTemplate(t *testing.T) {
+	params := DefaultParams()
+	res, err := getQueryFileAsBytes(`v004-go-tmpl-test.cql.tmpl`, params)
+	require.NoError(t, err)
+
+	queryStrings, err := getQueriesFromBytes(res)
+	require.NoError(t, err)
+
+	expOutputQueries := []string{
+		`CREATE TYPE IF NOT EXISTS jaeger_v1_dc1.keyvalue (
+key             text,
```

Comment:
all these details are not relevant for the unit test - after all we don't know if they are correct anyway since we're not testing against a real DB (but the integration test will test against it). So at most you can verify that CREATE commands are issued for expected db objects, but there's no need to match full queries.

Reply:
Removed this test. Now, we have an integration test for it: `bash scripts/cassandra-integration-test.sh -s 4 v004 v2`